T/TetGen/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+using BinaryBuilder
+
+name = "TetGen"
+version = v"1.5.1"
+
+#
+# Artifact builder for TetGen (c) Hang Si, see project home page https://tetgen.org
+# TetGen's license is  AGPLv3.
+#
+# TetGen source is C++ code, interfacing to Julia works via C wrapper by Simon Danisch
+# in the cwrapper subdirectory.
+#
+
+#
+# For 1.5.1 use the same upstream  source as in tetgenbuilder
+# Tentative upstream source for 1.6:
+# "http://www.tetgen.org/1.5/src/tetgen1.6.0.zip" => "e7bbbb4fb8f47f0adc3b46b26ab172557ebb90808c06e21b902b2166717af582"
+sources = [
+    GitSource("https://github.com/ufz/tetgen.git","9c4b70d479c6f67cb9e16dbe65f81ec3b821acad"),
+    DirectorySource("cwrapper",target="cwrapper")
+]
+
+
+
+script = raw"""
+# This will be used for 1.6
+# zipname=tetgen1.6.0
+# cd $WORKSPACE/srcdir/$zipname
+
+libdir="lib"
+if [[ ${target} == *-mingw32 ]]; then     libdir="bin"; else     libdir="lib"; fi
+mkdir ${prefix}/${libdir}
```

Comment:
`${libdir}` is already defined: https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables
```suggestion
mkdir -p ${libdir}
```